tests/cross_functional/ui/test_create_pool_block_pool.py

```diff
@@ -143,3 +156,42 @@ def test_create_delete_pool(
             raise PoolNotReplicatedAsNeeded(
                 f"Pool {self.pool_name} not replicated to size {replica}"
             )
+
+        # Check pg_num and osd_pool_default_pg_num matches
+        ct_pod = get_ceph_tools_pod()
+        osd_pool_default_pg_num = ct_pod.exec_ceph_cmd(
+            ceph_cmd="ceph config get mon osd_pool_default_pg_num"
+        )
+        logger.info(f"The osd pool default pg num value is {osd_pool_default_pg_num}")
+        expected_pgs = {
+            self.pool_name: osd_pool_default_pg_num,
+        }
+        assert validate_num_of_pgs(
+            expected_pgs
+        ), "pg_num is not equal to the osd pool default pg num"
+        logger.info(
+            f"pg_num of the new pool {self.pool_name} "
+            f"is equal to the osd pool default pg num {osd_pool_default_pg_num}"
+        )
+
+        # Check if the pg-autoscale is ON
+        pool_autoscale_status = ct_pod.exec_ceph_cmd(
+            ceph_cmd="ceph osd pool autoscale-status"
+        )
+        for pool in pool_autoscale_status:
+            if pool["pool_name"] == self.pool_name:
+                assert pool["pg_autoscale_mode"] == "on", "PG autoscale mode is off"
+        logger.info(f"{self.pool_name} autoscale mode is on")
+
+        # Check the pool is not none
+        oc_obj = OCP(kind=constants.CEPHBLOCKPOOL)
+        cbp_output = run_cmd(
+            cmd=f"oc get cephblockpool/{self.pool_name} -n {constants.OPENSHIFT_STORAGE_NAMESPACE} -o yaml"
+        )
+        cbp_output = oc_obj.exec_oc_cmd(
+            command=f"get cephblockpool/{self.pool_name} -n {constants.OPENSHIFT_STORAGE_NAMESPACE} -o yaml"
```

Comment:
please replace constants.OPENSHIFT_STORAGE_NAMESPACE with config.ENV_DATA["cluster_namespace"] otherwise it will fail on deployment with non-default ns